cli/entry.go

```diff
@@ -49,20 +77,21 @@ func Start(hostPort string, password string, in io.Reader, out io.Writer) {
 			if err == io.EOF {
 				return
 			}
-			fmt.Fprintln(os.Stderr, "Failed to read command:", err.Error())
+			_, _ = fmt.Fprintln(os.Stderr, "Failed to read command:", err.Error())
 			continue
 		}
 
 		if reqId != respReqId {
-			fmt.Fprintln(out, "Weird. This response is for another request.")
+			_, _ = fmt.Fprintln(out, "Weird. This response is for another request.")
 		}
 
-		fmt.Fprintln(out, resp)
-		out.Write([]byte("> "))
+		resp = colorize(resp)
+		_, _ = fmt.Fprintln(out, resp)
+		_, _ = out.Write([]byte("> "))
```

Comment:
Thank you. That one has been annoying me and I didn't think about such an elegant approach.

Reply:
No problem :) Love your docker image, man!